src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1653,6 +1653,21 @@ End Class
     End Sub
 
     <Fact>
+    Public Sub PublicSign_NoKey()
```

Comment:
Maybe I missed it but i don't see a similar test for C# here.  

What is the justification for having `/publicsign` without a key be functional?  Seems like it should be an error for sure.  I asked for something and the compiler is not doing it.